File: src/HeisenslaughtUI/src/app/services/draft-config.ts
export interface IDraftConfig {
    firstPick: number;
    pickTime: number;
    bonusTime: number;
    bankTime: boolean;
    team1Name: string;
    team2Name: string;
    mapName: string;
    disabledHeroes?: string[];
}



export interface ICreateDraftResult extends IDraftConfig {
    draftToken: string;
    team1DrafterToken: string;
    team2DrafterToken: string;
}

File: src/Heisenslaught/Infrastructure/DraftConfig.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;


namespace Heisenslaught.Infrastructure
{
    public class DraftConfig
    {
        public int firstPick;
        public int pickTime;
        public int bonusTime;
        public bool bankTime;
        public string team1Name;
        public string team2Name;
        public string map;
        public List<string> disabledHeroes;
        public DraftState state;
    }
}
